.github/workflows/nightly.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         with:
           ref: "${{ github.event.inputs.rev }}"
       - name: Run the container to execute the coverage script
-        run: docker run -v $(pwd):/tmp/parsec -w /tmp/parsec --security-opt seccomp=unconfined ghcr.io/parallaxsecond/parsec-service-test-all /tmp/parsec/ci.sh coverage
+        run: docker run -v $(pwd):/tmp/parsec -w /tmp/parsec --security-opt seccomp=unconfined --env RUST_TOOLCHAIN_VERSION=1.66.0 ghcr.io/parallaxsecond/parsec-service-test-all /tmp/parsec/ci.sh coverage
```

Comment:
As above, we could have this as an input/jenkins parameter instead of hardcoded

Reply:
Here we are fine with just running with MSRV, I don't think there's much benefit on being able to run it with a different compiler.
To avoid hardcoding certain parts of the command, we could use https://github.com/parallaxsecond/parsec/blob/main/.github/actions/ci_script/action.yml, and add an input for docker flags in there. This could be for a different PR as I don't think it's relevant for the topic of this one.